test/integration/tasksAPITests.coffee

```diff
@@ -326,8 +342,23 @@ describe "API Integration Tests", ->
     describe '*getTask(taskId)', ->
```

Comment:
It might be good to add some more tests for getTask: two examples I can think of are:
* Add a task with failed and completed transactions, and see if the filter only returns the failed transactions
* Add a task with >10 transactions and check that the returned number is 10 (with a limit turned specified)